### DISCUSSION TITLE
What are the ways to prevent miners from overfitting and acting maliciously?

### DISCUSSION THREAD

**Discussion by donbusha** [web-genie-ai/web-genie-ai#9] [Q&A] [answered]
As miners aim to maximize their incentives, they might overfit AI models using limited scope data over time.
How can we resolve this issue?

**Comment by sangar-1028** [comment]
To address that issue, we need to update our basic model with new datasets and make more synthetic data over time, so that miners cannot keep up with the updates.

**Comment by mrbreo** [comment, marked as answer]
It would be beneficial for us to utilize the validator model rotation methodology and the dataset expansion approach for generating synthetic data. Additionally, the best approach is to involve the miners themselves in the generation of synthetic data.

**Reply by donbusha** [reply]
How can we involve miners in the generation of synthetic data? It seems to surpass the Bittensor concept.

**Reply by mrbreo** [reply]
No, we can leverage Bittensor resources as much as possible. In this case, validators can generate synthetic questions along with the corresponding ground truth answers and send the questions to miners. Miners can then regenerate the questions and send them back. Finally, validators evaluate the questions and transfer the best ones to the miners.

**Reply by sangar-1028** [reply]
Awesome!!!
